firmware/quadruna/VC/src/app/states/app_driveWarningState.c

```diff
@@ -0,0 +1,57 @@
+#include "app_allStates.h"
+#include "app_driveState.h"
+
+#include "app_utils.h"
+#include "app_canAlerts.h"
+#include "app_canTx.h"
+
+static void driveWarningStateRunOnEntry(void)
+{
+    app_canTx_VC_State_set(VC_DRIVE_WARNING_STATE);
+
+    app_canTx_VC_LeftInverterEnable_set(true);
+    app_canTx_VC_RightInverterEnable_set(true);
+    app_canTx_VC_LeftInverterTorqueCommand_set(0.0f);
+    app_canTx_VC_RightInverterTorqueCommand_set(0.0f);
+    app_canTx_VC_LeftInverterTorqueLimit_set(0.0f);
+    app_canTx_VC_RightInverterTorqueLimit_set(0.0f);
+    app_canTx_VC_LeftInverterDirectionCommand_set(INVERTER_REVERSE_DIRECTION);
+    app_canTx_VC_RightInverterDirectionCommand_set(INVERTER_FORWARD_DIRECTION);
+}
+
+static void driveWarningStateRunOnTick1Hz(void)
+{
+    app_allStates_runOnTick1Hz();
+}
+
+static void driveWarningStateRunOnTick100Hz(void)
```

Comment:
This is really good. One note that this gonna work perfectly with the new fault/warning/alerts combination but I fear that if we run this on the quadruna right now we'll always be in warning state since we are getting a lot of warnings LMAOO

Reply:
This is just something for us to remember/note down